### PR TITLE
added a git question

### PR DESCRIPTION
**Pull request by lakshyabhasin509**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/git-quiz.ts [hunk 1]
```diff
@@ -366,6 +366,16 @@ const gitQuiz = [
...
   },
+  {
+    Question: "In Git, what command is used to roll back Git code to a previous commit?",
+    Answer: "git reset",
+    Distractor1: "git revert",
+    Distractor2: "git rebase",
+    Distractor3: "git reflog",
+    Explanation:
+      "In the command line, to roll Git code to a previous commit can run git reset with the hash of the commit you want to roll back to. There are three mode for the reset command --soft , --hard",
+    Link: "https://www.freecodecamp.org/news/the-ultimate-guide-to-git-reset-and-git-revert/#:~:text=The%20git%20reset%20command%20allows,or%20another%20remote%20repository%20yet.",
```
Comment (jdwilkin4) [inline src/data/git-quiz.ts:377]:
```suggestion
    Link: "https://www.freecodecamp.org/news/the-ultimate-guide-to-git-reset-and-git-revert/",
```

File: src/data/git-quiz.ts [hunk 2]
```diff
@@ -366,6 +366,16 @@ const gitQuiz = [
       "In the command line, to undo unwanted changes you can run git revert with the hash of the commit you want to revert back to.",
     Link: "https://www.freecodecamp.org/news/10-important-git-commands-that-every-developer-should-know/",
   },
+  {
+    Question: "In Git, what command is used to roll back Git code to a previous commit?",
```
Comment (jdwilkin4) [inline src/data/git-quiz.ts:370]:
```suggestion
    Question: "In Git, which command is used to reset a branch to a previous commit?",
```

File: src/data/git-quiz.ts [hunk 3]
```diff
@@ -366,6 +366,16 @@ const gitQuiz = [
...
+    Answer: "git reset",
+    Distractor1: "git revert",
+    Distractor2: "git rebase",
+    Distractor3: "git reflog",
+    Explanation:
+      "In the command line, to roll Git code to a previous commit can run git reset with the hash of the commit you want to roll back to. There are three mode for the reset command --soft , --hard",
```
Comment (jdwilkin4) [inline src/data/git-quiz.ts:376]:
```suggestion
      "The git reset command is used to reset your current HEAD to a specified state. ",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by lakshyabhasin509** [comment]
Did the specified changes.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**